third_party/models/BusterDrone/scene.gltf

```diff
@@ -9101,7 +9101,10 @@
         0,
         0,
         0
-      ]
+      ],
+      "extras": {
```

Comment:
Please don't change this model. It's not ours and we don't want to fork it.

Reply:
Reasonable point.